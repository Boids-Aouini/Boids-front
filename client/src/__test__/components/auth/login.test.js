import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';
import Login from '../../../components/auth/login';
import ReactDOM from 'react-dom';

const storeMock = configureStore();
let intialTestState = {
    auth: {
        openedAcc: null
    }
}
let store = storeMock(intialTestState)
describe('<Login />', () => {
    describe('render()', () => {
        test('renders without crashing', () => {
            shallow(<Login store={store} />);
        });

        test('renders component', () => {
            const wrapper = shallow(<Login store={store} />);
            const component = wrapper.dive();

            expect(toJson(component)).toMatchSnapshot();
        })
    });
});