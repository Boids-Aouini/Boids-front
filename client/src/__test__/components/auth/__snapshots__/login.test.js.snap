// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> render() renders component 1`] = `
<Login
  auth={
    Object {
      "openedAcc": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaWF0IjoxNTg5MjczNjM1fQ.CjLls-VQdsfjlPWdLueg9JHR0VWtz0swUdgYiQ-cESI",
    }
  }
  loginAction={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
